bundles/org.openhab.binding.sleepiq/src/main/java/org/openhab/binding/sleepiq/internal/SleepIQBindingConstants.java

```diff
@@ -59,6 +59,50 @@ public class SleepIQBindingConstants {
     public static final String CHANNEL_LEFT_ALERT_DETAILED_MESSAGE = "left#alertDetailedMessage";
     public static final String CHANNEL_RIGHT_ALERT_DETAILED_MESSAGE = "right#alertDetailedMessage";
 
+    // public static final String CHANNEL_BED_FIRST_NAME = "bed#firstName";
```

Comment:
Left-over? Same for other commented out lines below.

Reply:
The original author added constants as placeholders for a single-chamber bed, which there was no one with a single-chamber bed to test. I continued that convention by adding constants for the new channels I added. There are constants in this source file, and there's a thing-type in the things.xml for a single-chamber bed. I would be fine leaving them all, or removing them all. Just let me know which you would prefer. (I know the general rule is to not leave this stuff in the code, especially considering it's easy to add back in later).

Reply:
@jlaur What do you want to do about this?  Should I delete the commented constants and the single-chamber bed thing type?

Reply:
I would probably get rid of the constants here, since they are unused and don't provide much value. On the other hand, I would leave the commented definitions in the XML file as-is, since they document the intention for introducing this at some later time.

Reply:
Fair enough.